package.json

```diff
@@ -13,8 +13,7 @@
   ],
   "scripts": {
     "build": "wp-scripts build --webpack-copy-php --webpack-src-dir=blocks",
-    "dev": "npm run start",
```

Comment:
Wonder if we should keep `dev` around so that folks have a consistent experience when starting a project's development environment. `npm run dev` is what I reach to first when starting up at least

Reply:
yeah, good point. I saw it and was just trying to clean up. But that certainly makes sense.